roles/matrix-bot-matrix-registration-bot/templates/env.j2

```diff
@@ -0,0 +1,8 @@
+BOT_SERVER={{ matrix_bot_matrix_registration_bot_bot_server }}
+BOT_USERNAME={{ matrix_bot_matrix_registration_bot_matrix_user_id_localpart }}
+BOT_ACCESS_TOKEN={{ matrix_bot_matrix_registration_bot_api_token }}
+API_BASE_URL={{ matrix_bot_matrix_registration_bot_api_base_url }}
+API_TOKEN={{ matrix_bot_matrix_registration_bot_matrix_admin_token }}
```

Comment:
This should be called `matrix_bot_matrix_registration_bot_api_token`.

Reply:
Removed the file but changed the variable name accordingly